src/interfaces/CheckoutSuccessType.ts

```diff
@@ -0,0 +1,13 @@
+export type CheckoutSuccessType = {
```

Comment:
The naming is a bit vague and "Type" shouldn't be in there. Maybe `CheckoutSuccessResult`?